app.py

```diff
@@ -0,0 +1,53 @@
+from flask import Flask, jsonify, request, make_response
+from datetime import datetime
+
+app = Flask(__name__)
+
+entries = [
```

Comment:
Would appreciate if you could use OOP
Look at the book example we did and think of how you can leverage OOP to solve this

Reply:
I will look at it tomorrow